### ISSUE TITLE
[cocos2dx] Two tint colour rendering issue

### ISSUE THREAD

**Issue by Wizermil** [EsotericSoftware/spine-runtimes#1229]
Hey I think after the refactoring of cocos2dx runtime you introduced a bug in the rendering of spines using two tint colour.

Here is the result of the export from the editor
![export_spine](https://user-images.githubusercontent.com/700192/50006335-bec7f280-ffad-11e8-9985-8e52ddf49ae2.png)
and the output from the simulator:
![screenshot_simulator](https://user-images.githubusercontent.com/700192/50006496-33029600-ffae-11e8-8380-fdd306d6f644.png)
As you can see the rendering of the colours is not the one we were expecting. After replacing
```cpp
darkColor.r = red * slot->darkColor->r;
darkColor.g = green * slot->darkColor->g;
darkColor.b = blue * slot->darkColor->b;
```
in https://github.com/EsotericSoftware/spine-runtimes/blob/3.6/spine-cocos2dx/src/spine/SkeletonRenderer.cpp#L418 by
```cpp
darkColor.r = 255 * slot->darkColor->r;
darkColor.g = 255 * slot->darkColor->g;
darkColor.b = 255 * slot->darkColor->b;
```
it's back to normal
![screenshot_simulator_pass](https://user-images.githubusercontent.com/700192/50006830-5548e380-ffaf-11e8-957f-59f339e8d67c.png)
however I don't know if this modification introduces other issues. Do you have a test suite that I could use to check?

PS: I attached the spine I used for my test 
[debug_two_color_tint_additive_alpha.zip](https://github.com/EsotericSoftware/spine-runtimes/files/2680250/debug_two_color_tint_additive_alpha.zip)

**Comment by badlogic** [comment]
Thanks for the very helpful report! You also already identified the issue correctly. I'll get to this asap.

**Comment by badlogic** [comment]
This has been fixed on the 3.6 and 3.7-beta-cpp branches. The 3.7-beta branch was not affected. Thanks for reporting!
